server/src/main/java/org/apache/druid/client/selector/TierSelectorStrategy.java

```diff
@@ -47,6 +49,22 @@
   List<QueryableDruidServer> pick(
```

Comment:
Any reason to keep the old interfaces? If this is for backward-compatibility, it maybe make sense. But please annotate them as `@Deprecated`.

Reply:
I marked the non-query ones as deprecated throughout